### PR TITLE
Feature/d 10 prepare

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/LOOP-1276

- Update contrib modules
- Update custom modules
- Update config to match module update.
- Change twig {% spaceless %}
- Change jquery once.
- Disable samlauth

### CODE REVIEW

File: web/profiles/custom/os2loop/modules/os2loop_question/assets/js/hideReferenceIds.js [hunk 1]
```diff
@@ -9,33 +8,32 @@
     Drupal.behaviors.autocompleteReferenceEntityId = {
       attach: function (context) {
         // Remove reference IDs for autocomplete elements on init.
-        $('.form-autocomplete', context).once('replaceReferenceIdOnInit').each(function () {
+        $(once('replaceReferenceIdOnInit', '.form-autocomplete', context)).each(function () {
```
Comment (rimi-itk) [inline web/profiles/custom/os2loop/modules/os2loop_question/assets/js/hideReferenceIds.js:11]: This looks weird. Is the `once` function suddenly defined in the global scope?

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good (I haven't reads all the config files, though). A single question asked.

File: web/profiles/custom/os2loop/modules/os2loop_question/assets/js/hideReferenceIds.js (hunk 1 above)
Comment (martinyde) [reply web/profiles/custom/os2loop/modules/os2loop_question/assets/js/hideReferenceIds.js:11]: Added core/once as a library dependency and in wrapper

Comment (martinyde) [reply web/profiles/custom/os2loop/modules/os2loop_question/assets/js/hideReferenceIds.js:11]: https://www.drupal.org/node/3158256

Comment (rimi-itk) [reply web/profiles/custom/os2loop/modules/os2loop_question/assets/js/hideReferenceIds.js:11]: I'm pretty I don't understand what `$(once(…))` actually does, but I trust you have tested it and that it works.

**Review by rimi-itk — APPROVED** [review]

**Pull request closed without merging**